pg_lakehouse/src/fdw/base.rs

```diff
@@ -207,6 +216,59 @@ async fn create_delta_provider(
     Ok(Arc::new(delta_table) as Arc<dyn TableProvider>)
 }
 
+#[inline]
+async fn create_iceberg_provider(
+    table_options: HashMap<String, String>,
+) -> Result<Arc<dyn TableProvider>, BaseFdwError> {
+    let path = require_option(TableOption::Path.as_str(), &table_options)?;
+
+    // TODO: Replace with non-REST.
+    let config = GlueCatalogConfig::builder()
```

Comment:
Glue is the AWS catalog provider, no? Does this mean this only works with S3 storage?

Reply:
`iceberg-rust` only has Glue, Rest, And Hive catalogs implemented, so yes, that seems like it'll be a limitation for now.